ereolen_support/templates/ereolen-support--lead-text.tpl.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * @file
+ * Template for showing form lead text.
+ */
+?>
+<div class="form-item">
+  <?php $text = isset(variable_get('ereolen_support_lead')['value']) ? variable_get('ereolen_support_lead')['value'] : '' ?>
```

Comment:
Use `variable_get('ereolen_support_lead')['value'] ?? NULL` (or similar) here – or pass the value as a theme variable.